src/Views/Display.php

```diff
@@ -71,7 +71,7 @@ public function get(): Component|array
         $max_width = (string) ($this->properties[\ilOpencastPageComponentPluginGUI::PROP_WIDTH] ?? '');
         $max_width = $max_width !== '' ? $max_width . 'px' : 'auto';
 
-        $configured_aspect_ratio =  ((float) $this->properties[\ilOpencastPageComponentPluginGUI::PROP_ASPECT_RATIO]) ?? Edit::RATIO_AS_PUBLICATION);
+        $configured_aspect_ratio = (float) (($this->properties[\ilOpencastPageComponentPluginGUI::PROP_ASPECT_RATIO]) ?? Edit::RATIO_AS_PUBLICATION);
```

Comment:
With this change, we went back to the initial issue of having the `$configured_aspect_ratio` always as float!
This will affect the next comparison of `===`.
So this needs to be:
```suggestion
        $configured_aspect_ratio = ((float) $this->properties[\ilOpencastPageComponentPluginGUI::PROP_ASPECT_RATIO]) ?? Edit::RATIO_AS_PUBLICATION;

```

Reply:
This change is invalid an reintroduces the missing unclosed { error. I will update the commit in another  way

Reply:
Since it is a suggestion, therefore, please adjust it in the IDE, the main goal is to highlight the outcome!

Reply:
resolved in https://github.com/opencast-ilias/OpencastPageComponent/pull/38/commits/3973ce681155335028c8cfdaf0c9f6264642b697